standards/documenting.md

```diff
@@ -30,13 +30,14 @@ order: 8
 
 ## How to test
 
-* test readability of documentation with other stakeholders, professionals from other public organisations and the general public
-* generate documentation from code
-* test links and images
+* Other stakeholders, professionals from other public organisations and the general public find the documentation clear and understandable
```

Comment:
I feel this one is a bit hard to continously test, is there a way we could store this information so that it can be referenced at every check, otherwise I feel we might need to solve this differently